pkg/controller/controller_broker_test.go

```diff
@@ -54,77 +54,82 @@ func TestShouldReconcileServiceBroker(t *testing.T) {
 	// name: short description of the test
 	// broker: broker object to test
 	// now: what time the interval is calculated with respect to interval
-	// internal: the time that has elapsed since now
 	// reconcile: whether or not the reconciler should run, the return of
 	// shouldReconcileServiceBroker
+	// ERIK:TODO: Test new Relist codepaths
```

Comment:
Looks like this TODO still needs to be done, accurate?

Reply:
Have a local commit with a bunch of error handling I wanted to run by you. It tests the RelistDuration == nil path, but I want to make sure I hit the rest of them.